include/zephyr/bluetooth/mesh/brg_cfg.h

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_INCLUDE_BLUETOOTH_MESH_BRG_CFG_H__
+#define ZEPHYR_INCLUDE_BLUETOOTH_MESH_BRG_CFG_H__
+
+#include <stdint.h>
+#include <stdbool.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @defgroup bt_mesh_brg_cfg Bridge Configuration common header
+ * @ingroup bt_mesh
+ * @{
+ */
+
+#define BT_MESH_SUBNET_BRIDGE_DISABLED 0x00
+#define BT_MESH_SUBNET_BRIDGE_ENABLED  0x01
+
+/** Bridging Table state entry corresponding to a entry in the Bridging Table. */
+struct bridging_table_entry {
+	/** Allowed directions for the bridged traffic (or bridged traffic not allowed) */
+	uint32_t directions:8;
+	/** NetKey Index of the first subnet */
+	uint32_t net_idx_1:12;
+	/** NetKey Index of the second subnet */
+	uint32_t net_idx_2:12;
+	/** Address of the node in the first subnet */
+	uint16_t addr_1;
+	/** Address of the node in the second subnet */
+	uint16_t addr_2;
+} __packed;
```

Comment:
I strictly disagree to use packed structures in public API. 
1. It adds significant limitation for usage of it. Users have to consider it to avoid nonalignment access faults. Limitation in pointers usage due to typecasting causes loosing of attributes.
2. Compiler generates extremely not optimal code to avoid potential alignment issue. User looses advantages to use 32 (and wider) platforms here.